main.gms

```diff
@@ -278,10 +278,10 @@ cm_LimRock             "limit amount of rock spread each year [Gt]"
 c_tau_so2_xmpt       "switch for temporarily (mainly in the past) exempting chinese SO2 emissions from the SO2 tax"
 cm_expoLinear_yearStart "time at which carbon price increases lineraly instead of exponentially"
 
-c_budgetCO2FFI        "carbon budget for CO2 emissions from FFI (in GtCO2)"
+c_budgetCO2FFI        "carbon budget for CO2 emissions starting from 2020 from FFI (in GtCO2)"
 c_abtrdy              "first year in which advanced bio-energy technology are ready (unit is year; e.g. 2050)"
 c_abtcst              "scaling of the cost of advanced bio-energy technologies (no unit, 50% increase means 1.5)"
-c_budgetCO2        "carbon budget for all CO2 emissions (in GtCO2)"
```

Comment:
we may want to change the default values for c_budgetCO2 here as well to 1150

Reply:
Also in the danger zone in main.gms? 😮